maths/find_max.py

```diff
@@ -1,16 +1,21 @@
-# NguyenU
```

Comment:
This file is out of sync with `master`.  There are already doctests on this file:
https://github.com/TheAlgorithms/Python/blob/master/maths/find_max.py#L6

Reply:
@cclauss Please review again!